lib/util/search/address_search_delegate.dart

```diff
@@ -38,7 +38,7 @@ class AddressSearchDelegate extends SearchDelegate<AddressSuggestion?> {
   // [bool mounted = true] is a hack to be able to use context
   // (StatelessWidget is always mounted, so this is fine)
   void returnFirstResult(BuildContext context, [bool mounted = true]) async {
-    final List<AddressSuggestion> suggestions = await addressSuggestionManager.getAddressSuggestions(query);
+    final List<AddressSuggestion> suggestions = await addressSuggestionManager.getSuggestions(query);
```

Comment:
fyi this is not immediately related, but I noticed that I forgot to change that when adding the other suggestion types. Of course we should return the overall first result, not the first address.